consensus/core/src/types.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    block::{Block, BlockAPI},
+    utils::{format_authority_index, format_authority_round},
+};
+
+use serde::{Deserialize, Serialize};
+use std::fmt::{self, Display};
+
+use consensus_config::AuthorityIndex;
+
+/// The consensus protocol operates in 'waves'. Each wave is composed of a leader
+/// round, at least one voting round, and one decision round.
+#[allow(unused)]
+pub type WaveNumber = u32;
+
+/// Round number of a block.
+pub type Round = u32;
```

Comment:
Round has been defined in block.rs

Reply:
I hope we can have more specific file names even when they just contain types. A single `types.rs` file for all types seems a bit too long. But we can see how this will work out.